### PR TITLE
create default menu item upon initialization; closes #541

### LINKED ISSUE #541: add some default links
Report a bug and feature suggestions
Ranks link

### PR DESCRIPTION
**Pull request by Dylan-RG-Patterson**
creates a menu item for the ranks list in the drop down menu upon tenant creation.
image

### CODE REVIEW

**Comment by tylerecouture** [comment]
@Dylan-RG-Patterson tests are failing.

File: src/tenant/tests/test_initialization.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 from badges.models import BadgeRarity
 from courses.models import MarkRange
+from utilities import MenuItem
```
Comment (tylerecouture) [inline src/tenant/tests/test_initialization.py:7]: did you run this test?  haha.  `from utilities.models import MenuItem` is prolly what you meant.  Please save me time reviewing and run tests and flake8 locally before pushing =)

**Review by tylerecouture — CHANGES_REQUESTED** [review]

File: src/tenant/tests/test_initialization.py (hunk 1 above)
Comment (Dylan-RG-Patterson) [reply src/tenant/tests/test_initialization.py]: Sorry, had to rewrite that section, did it too hastily and forwent the necessary procedures. Bad import has been fixed.

File: src/tenant/initialization.py [hunk 2]
```diff
@@ -212,6 +214,17 @@ def create_initial_badges():
     ])
 
 
+def create_initial_menu_items():
+    MenuItem.objects.create(
+        label="Ranks List",
+        fa_icon="star-o",
+        url="/courses/ranks/",
```
Comment (tylerecouture) [inline src/tenant/initialization.py:221]: this shouldn't be hard coded, it should use `reverse('courses:ranks_list')` (or whatever that view is called) so it looks up the path in urls.py

If that's not possible (circular imports or some reason) Then need to add a test that this menu item's url is accessible (back up plan, reverse is better)

**Review by tylerecouture — CHANGES_REQUESTED** [review]
Request small tweak

File: src/tenant/initialization.py (hunk 2 above)
Comment (Dylan-RG-Patterson) [reply src/tenant/initialization.py]: url changed from hard-coded to reverse url.

**Review by tylerecouture — APPROVED** [review]

**Pull request merged**